content/attacks/posts/2023-04-02-Allbridge.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2023-04-02
+target-entities: 
+- Allbridge
+entity-types:
+- DeFi
+- Bridge
+attack-types:
+- Smart Contract Exploit
+- Flash Loan Attack
+title: "A Close Look into the $573K Allbridge Exploit and Its Resolution"
+---
+
+## Summary
+
+On April 2, 2023, Allbridge, a cross-chain bridging protocol, experienced a flash loan exploit that drained the liquidity pools of stablecoins USDT and BUSD on the Binance Smart Chain platform. As a result of this exploit, hackers made off with an estimated $573K.
+
+## Attackers
+
+The identity of the attacker is unknown.
+
+BSC:
+- [0xC578d755Cd56255d3fF6E92E1B6371bA945e3984](https://bscscan.com/address/0xc578d755cd56255d3ff6e92e1b6371ba945e3984)
```

Comment:
Can't find relation to the bridge hack involving this address. Could you be able to attach a source in the comments?